peep.py

```diff
@@ -23,6 +23,14 @@
 # all copies or substantial portions of the Software.
 from __future__ import print_function
 try:
+    reload
+    import sys
+    reload(sys)
```

Comment:
reload() scares me for various reasons. (For example, symbols defined in the previously loaded incarnation of the module will be duplicated, leading to fun behaviors like previously created instances of classes starting to fail isinstance() checks.) Can you explain why reload() is necessary here and whether there are any alternatives?